staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -585,6 +593,7 @@ type JobStatus struct {
 
 	// The number of active pods which have a Ready condition and are not
 	// terminating (without a deletionTimestamp).
+	// +optional
```

Comment:
Only validated when not nil
https://github.com/kubernetes/kubernetes/blob/e34f01402f2be14e68b03a58c054f1a86bff55b9/pkg/apis/batch/validation/validation.go#L465-L467
Agreed optional